scripts/change-log/Cargo.toml

```
@@ -0,0 +1,10 @@
+[package]
+name = "change-log"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+regex = "1"
+dotenv = "0.15"
+tokio = { version = "1", features = ["full"] }
+octocrab = "0.39"
```

Makes the script use exactly the same versions as all our other crates where possible, potentially saving compilation time:

```suggestion
[dependencies]
regex = { workspace = true }
dotenv = "0.15"
tokio = { workspace = true, features = ["full"] }
octocrab = "0.39"
```